translationCoreApps/ScripturePane
# Refactor Modal to Render by Rows not Columns

#### This pull request addresses:

Scripture pane now keeps verses in Sync.


#### How to test this pull request:

Ensure Scripture pane keeps verses in sync and maintains all functionality.
Swap projects and tools, test directional targets LTR/RTL, and Greek pop-over functionality.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/scripturepane/52)
